### PR TITLE
Adds autoComplete to form.

### PR DESCRIPTION
**Pull request by crwallace**
This PR adds the `autoComplete` attribute to the `form` element to allow users to autofill when supported by a browser.

### CODE REVIEW

File: src/client/components/form-elements/ErrorableSelect.jsx [hunk 1]
```diff
@@ -86,6 +88,8 @@ class ErrorableSelect extends React.Component {
 
 ErrorableSelect.propTypes = {
   errorMessage: React.PropTypes.string,
+  name: React.PropTypes.string,
+  autocomplete: React.PropTypes.string,
```
Comment (akainic) [inline src/client/components/form-elements/ErrorableSelect.jsx]: Sorry this is really tedious, but could we also add these props in the description at the top of the file in the comments? For all the files in this PR you've touched?

File: src/client/components/HealthCareApp.jsx [hunk 2]
```diff
@@ -148,10 +148,11 @@ class HealthCareApp extends React.Component {
         </div>
         <div className="medium-8 columns">
           <div className="progress-box">
-            <div className="form-panel">
+          {/* TODO: Change action to reflect actual action for form submission. */}
+            <form className="form-panel" autoComplete="on">
```
Comment (akainic) [inline src/client/components/HealthCareApp.jsx]: Do we need to specify `autocomplete="on"` for this to work? I think `on` is the default.

**Comment by akainic** [comment]
LGTM with a couple nits!

File: src/client/components/form-elements/ErrorableSelect.jsx (hunk 1 above)
Comment (crwallace) [inline src/client/components/form-elements/ErrorableSelect.jsx]: Will do!

File: src/client/components/HealthCareApp.jsx (hunk 2 above)
Comment (crwallace) [inline src/client/components/HealthCareApp.jsx]: I thought so too but for some reason it was not working at first. I will remove and test this.

**Pull request merged**